column/api/backend/cache/local_mem.py

```diff
@@ -0,0 +1,51 @@
+from collections import deque  # noqa
+import threading
+
+import store
+
+
+# Shared dictionaries for LocalMemoryCache
+_stores = {}
```

Comment:
Just a thought - move these variables to LocalMemoryStore or store.Store as class static members?